pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -178,38 +177,79 @@ func (k *Deployer) Render(ctx context.Context, out io.Writer, builds []build.Art
 // adding image digests, and adding run-id labels.
 func (k *Deployer) renderManifests(ctx context.Context, _ io.Writer, builds []build.Artifact,
 	flags []string) (manifest.ManifestList, error) {
+	var err error
 	debugHelpersRegistry, err := config.GetDebugHelpersRegistry(k.globalConfig)
 	if err != nil {
 		return nil, fmt.Errorf("retrieving debug helpers registry: %w", err)
 	}
-	if k.Fn.SinkDir == "" {
```

Comment:
If `k.Fn.SinkDir` is not set, what would the directory the rendered manifests would be sinked to?

Reply:
It won't sink to any directory. Users only want to specify the sink dir when they plan to do further git operations on the hydrated manifests (e.g. store it in another repo).

Reply:
ah got it. does it make sense to remove `tmpSinkDir` from elsewhere?
Looks like it not required now.